app/code/Magento/Review/Controller/Product/Post.php

```diff
@@ -31,6 +31,13 @@ public function execute()
             return $resultRedirect;
         }
 
+        $isReviewEnable = $this->_objectManager->get(\Magento\Review\Helper\Data::class)->isEnableReview();
```

Comment:
Please don't use Object manager, use constructor dependency injection instead.
Also please don't extend existing helpers with new functionality. If you want to decompose class, use services instead.
In this particular controller, I'm sure you can use just private class.

Reply:
Also please add `declare(strict_types=1);` right after copyright.